app/models/stream.rb

```diff
@@ -1,16 +1,38 @@
 # frozen_string_literal: true
 
-#
-# Streams model.
-#
-class Stream
-  def self.data
-    JSON.parse(
-      File.read(File.join(File.dirname(__FILE__), '../../data/seedFile.json'))
-    )
-  end
+module Models
+  #
+  # Streams model.
+  #
+  class Stream
+    private_class_method def self.save(data)
```

Comment:
we could have something like below as the structure for a class which is widely adopted structure

```ruby

class Foo
  def self.bar; 1; end

  def self.access_baz; baz; end

  class << self
    private

    def baz; 2 end
  end
end
```